pkgs/development/libraries/python-qt/default.nix

```diff
@@ -50,6 +50,8 @@ stdenv.mkDerivation (finalAttrs: {
     "PYTHON_VERSION=3.${python3.sourceVersion.minor}"
   ];
 
+  makeFlags = lib.optional stdenv.isDarwin "LDFLAGS=-Wl,-install_name,$(out)/lib/libPythonQt-Qt5-Python${python3.version}.dylib";
```

Comment:
```suggestion
  makeFlags = lib.optionals stdenv.isDarwin [
    "LDFLAGS=-Wl,-install_name,$(out)/lib/libPythonQt-Qt5-Python${python3.version}.dylib"
  ];
```

Reply:
Thanks for the help, I'm totally lost here 😅 
But this is still not working for another package which depends on python-qt, and is not yet in nixpkgs:
```
> 1/1 Test #1: nodes ............................Subprocess aborted***Exception:   0.41 sec
       > dyld[93368]: Library not loaded: libPythonQt-Qt5-Python3.11.3.dylib
       >   Referenced from: <AE2EB1A4-62B5-361B-9879-B5E9EA63EC35> /private/tmp/nix-build-gepetto-viewer-5.0.0.drv-0/source/build/tests/nodes
       >   Reason: tried: '/libPythonQt-Qt5-Python3.11.3.dylib' (no such file), 'libPythonQt-Qt5-Python3.11.3.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OSlibPythonQt-Qt5-Python3.11.3.dylib' (no such file), 'libPythonQt-Qt5-Python3.11.3.dylib' (no such file), '/usr/local/lib/libPythonQt-Qt5-Python3.11.3.dylib' (no such file), '/usr/lib/libPythonQt-Qt5-Python3.11.3.dylib' (no such file, not in dyld cache), '/libPythonQt-Qt5-Python3.11.3.dylib' (no such file), '/private/tmp/nix-build-gepetto-viewer-5.0.0.drv-0/source/build/tests/libPythonQt-Qt5-Python3.11.3.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/private/tmp/nix-build-gepetto-viewer-5.0.0.drv-0/source/build/tests/libPythonQt-Qt5-Python3.11.3.dylib' (no such file), '/private/tmp/nix-build-gepetto-viewer-5.0.0.drv-0/source/build/tests/libPythonQt-Qt5-Python3.11.3.dylib' (no such file), '/usr/local/lib/libPythonQt-Qt5-Python3.11.3.dylib' (no such file), '/usr/lib/libPythonQt-Qt5-Python3.11.3.dylib' (no such file, not in dyld cache)t
 ```
 So I'm trying to do this in fixup instead:
 ```
python-qt> Running phase: fixupPhase
python-qt> + otool -D /nix/store/81nzbhbabxhjyb3qsdcs0qc49wr6iz5k-python-qt-3.5.0/lib/libPythonQt-Qt5-Python3.11.3.dylib
python-qt> /nix/store/81nzbhbabxhjyb3qsdcs0qc49wr6iz5k-python-qt-3.5.0/lib/libPythonQt-Qt5-Python3.11.3.dylib:
python-qt> libPythonQt-Qt5-Python3.11.3.dylib
python-qt> + install_name_tool -change libPythonQt-Qt5-Python3.11.3.dylib /nix/store/81nzbhbabxhjyb3qsdcs0qc49wr6iz5k-python-qt-3.5.0/lib/libPythonQt-Qt5-Python3.11.3.dylib /nix/store/81nzbhbabxhjyb3qsdcs0qc49wr6iz5k-python-qt-3.5.0/lib/libPythonQt-Qt5-Python3.11.3.dylib
python-qt> + otool -D /nix/store/81nzbhbabxhjyb3qsdcs0qc49wr6iz5k-python-qt-3.5.0/lib/libPythonQt-Qt5-Python3.11.3.dylib
python-qt> /nix/store/81nzbhbabxhjyb3qsdcs0qc49wr6iz5k-python-qt-3.5.0/lib/libPythonQt-Qt5-Python3.11.3.dylib:
python-qt> libPythonQt-Qt5-Python3.11.3.dylib
```
but no lock so far. I'll keep trying :)

Reply:
I think the issue there in libPython is a nix specific issue, see

>  '/libPythonQt-Qt5-Python3.11.3.dylib' (no such file)

no file should be written in "/" folder in nixbld. So when I see path like /Library/Framework I see a nix build-time error. It should perhaps be "$out/Library/Framework" maybe... it may not be the issue, it seems a bit truncated

Reply:
Ok, that was `-id` 🎉